src/components/Product/SessionReplay/index.tsx

```diff
@@ -330,7 +330,7 @@ const comparison = [
             LogRocket: '1 month',
             Matomo: '24 months',
             FullStory: '1 month',
-            PostHog: 'Up to 3 months',
+            PostHog: 'Up to 12 months',
```

Comment:
should we say longer retention is in beta?

Reply:
It might not fit